DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/activity/activity.service.js

```diff
@@ -0,0 +1,40 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.activity')
+        .factory('ActivityService', activityService);
+
+    activityService.$inject = ['$q', '$http', 'ACTIVITY_CONSTANTS'];
+
+    function activityService($q, $http, constants) {
+        var defer;
```

Comment:
as I understand you can leave 

```js

(function () {
    'use strict';

    angular
        .module('app.activity')
        .factory('ActivityService', activityService);

    activityService.$inject = ['$http', 'ACTIVITY_CONSTANTS'];

    function activityService($http, constants) {
        function getExternalActivity(page) {
            return $http.get(constants.EXTERNAL_ACTIVITY_URL + page).then(successCb, errorCb);
        }

        function getInternalActivity() {
            return $http.get(constants.INTERNAL_ACTIVITY_URL).then(successCb, errorCb);
        }

        function successCb(response) {
            return response.data;
        }

        function errorCb(reason) {
            return reason;
        }

        return {
            'getExternalActivity': getExternalActivity,
            'getInternalActivity': getInternalActivity
        };
    }
})();

```

And it will just work the same)